CHAID/CHAID.py

```diff
@@ -102,7 +103,9 @@ class CHAIDNode(object):
         The dependent variable set
     """
     def __init__(self, choices=None, split_variable=None, chi=0,
-                 p=0, indices=None, node_id=0, parent=None, dep_v=None):
+                 p=0, indices=None, node_id=0, parent=None, dep_v=None,
+                 is_terminal=False
```

Comment:
It used to be like that, but it's not terminal_indices, it's indices as we get the respondents in all of the nodes. So this won't work